src/Services/LimitService.ts

```diff
@@ -0,0 +1,145 @@
+import { QueryFeature, QueryFeatureType } from "../Enums";
+import { IModelService, IQueryLimitConfig } from "../Interfaces";
+import ApiError from "../Exceptions/ApiError";
```

Comment:
```suggestion
import AxeError from "../Exceptions/AxeError";
```

There is no such file like `src/Exceptions/ApiError`, right?